.github/workflows/ruby.yml

```diff
@@ -37,11 +37,15 @@ jobs:
             blacklight_version: 'latest'
             bootstrap_version: ~> 4.0
             additional_engine_cart_rails_options: -a propshaft
+          - rails_version: '7.1.2'
```

Comment:
Curious if there's a reason to test on 7.1.2 or if we can jump to 7.1.3.4 now?

Reply:
I think we can.